pkg/asset/installconfig/openstack/validation/machinepool.go

```diff
@@ -33,21 +35,25 @@ func ValidateMachinePool(p *openstack.MachinePool, ci *CloudInfo, controlPlane b
 
 	// Validate Root Volumes
 	if p.RootVolume != nil {
+		// No need to check the storage in the Flavor since RootVolume is used
+		checkStorageFlavor = false
 		if p.RootVolume.Type == "" {
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("rootVolume").Child("type"), p.RootVolume.Type, "Volume type must be specified to use root volumes"))
 		}
-		if p.RootVolume.Size <= 0 {
-			allErrs = append(allErrs, field.Invalid(fldPath.Child("rootVolume").Child("size"), p.RootVolume.Size, "Volume size must be greater than zero to use root volumes"))
+		if p.RootVolume.Size <= minimumStorage {
```

Comment:
why `<=` instead of `<`? Isn't it OK to boot from a volume with size exactly 25GB?

(also below, "at least" instead of "greater than")

Reply:
that's what happen when coding late !